go.mod

```diff
@@ -46,6 +46,7 @@ require (
 require (
 	github.com/charmbracelet/bubbles v0.10.0
 	github.com/charmbracelet/lipgloss v0.4.0
+	github.com/minio/minio-go v6.0.14+incompatible
```

Comment:
Can you do `go mod tidy` ?

Reply:
you need to depend on `github.com/minio/minio-go/v7`  most probably you are depending on `github.com/minio/minio-go` in one of your imports @anjalshireesh